pkg/scheduler/metrics/resources/resources_test.go

```diff
@@ -415,18 +415,18 @@ func Test_podResourceCollector_CollectWithStability(t *testing.T) {
 						Containers: []v1.Container{
 							{Resources: v1.ResourceRequirements{
 								Requests: v1.ResourceList{"cpu": resource.MustParse("1")},
-								Limits:   v1.ResourceList{"cpu": resource.MustParse("2")},
+								Limits:   v1.ResourceList{"cpu": resource.MustParse("2"), "memory": resource.MustParse("1G")},
```

Comment:
It seems that the OP intended the metric to be calculated the way it was.

/hold